core/src/Cuda/Kokkos_Cuda_Team.hpp

```diff
@@ -290,7 +290,7 @@ class CudaTeamMember {
    */
   template <typename Type>
   KOKKOS_INLINE_FUNCTION Type team_scan(const Type& value) const {
-    return this->template team_scan<Type>(value, 0);
```

Comment:
In the end, this was triggered by a change we don't want. It's consistent to use `nullptr` here anyway.